### ISSUE TITLE
Rename color & manufacturer table

### ISSUE THREAD

**Issue by Eagle3386** [Eagle3386/aya-platform#14]
Implementing #3 requires two table renames:
```sql
RENAME TABLE aya_vehicles_colors TO aya_colors;
RENAME TABLE aya_vehicles_manufacturers TO aya_manufacturers;
```

**Comment by Eagle3386** [comment]
Done in test environment.

@ToxSox you need to execute both statements above prior to launching v3.16!

**Comment by ToxSox** [comment]
done

**Comment by ToxSox** [comment]
Undo rename of RENAME TABLE aya_vehicles_manufacturers TO aya_manufacturers;, because the software uses the old names.

**Comment by Eagle3386** [comment]
Will be fixed upon release of its containing milestone.

**Comment by Eagle3386** [comment]
Fixed by 7c77a2fbfa765be9aabfd80d4859a5debef730ef
